components/pipeline-service/base/rbac/openshift-pipelines/configmap-viewer.yaml

```diff
@@ -0,0 +1,33 @@
+---
+# Grant access to authenticated users to be able to view
+# openshift-pipelines configMaps. The user case involves
+# service accounts running automated tests being able to
+# discover the custom URL for the Konflux UI
```

Comment:
I really appreciate the "why" comment here. Since this is going to apply to all users, is there any risk in all users having read access to configmaps? I'm wondering if the the automated testing serviceaccounts have a shared group which this rolebinding can use as its subject instead of all authenticated users